ament_cmake_core/cmake/symlink_install/ament_cmake_symlink_install_targets.cmake

```diff
@@ -71,6 +71,10 @@ function(ament_cmake_symlink_install_targets)
       endif()
       # TODO consider using a generator expression instead
       # $<TARGET_FILE:target>
+      # Until them use the old policy.
```

Comment:
The comment suggests that this "changes" the policy which is not the case. Since the code set `2.8.3` as the minimum required version the old policy is already being used implicitly.

Can you please update the comment to state the reason for setting the policy explicitly more clearly. E.g.:

    # set the policy explicitly in order to avoid warning with newer CMake versions